.eslintrc.js

```diff
@@ -11,4 +13,13 @@ module.exports = {
     jest: true,
   },
   rules: {},
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        'no-unused-vars': 'off',
+        '@typescript-eslint/no-unused-vars': 'off',
+      },
+    },
+  ],
```

Comment:
`@typescript-eslint/no-unused-vars` rule was causing issues for imported types. see: https://github.com/typescript-eslint/typescript-eslint/issues/363

I've added this as per https://43081j.com/2019/02/using-eslint-with-typescript